Build failure with maven version 3.9.4

Can build fine with apache-maven-3.8.5, but after upgrading to 3.9.4 I get the following error.

> [ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:3.1.0:run (set executable flags.) on project appengine-java-sdk: You are using 'tasks' which has been removed from the maven-antrun-plugin. Please use 'target' and refer to the >>Major Version Upgrade to version 3.0.0<< on the plugin site. -> [Help 1]

--- a/sdk_assembly/pom.xml
+++ b/sdk_assembly/pom.xml
@@ -276,11 +276,9 @@
                         <configuration>
                             <target>
                                 <chmod dir="${assembly-directory}/bin" perm="755" includes="**/*.sh" />
-                            </target>
-                            <tasks>
                                 <delete file="${assembly-directory}/jetty94/jetty-home/lib/apache-jsp/org.eclipse.jetty.apache-jsp-${jetty.version}.jar"/>
                                 <delete dir="${assembly-directory}/jetty94/jetty-home/demo-base"/>
-                            </tasks>
+                            </target>
                         </configuration>
                     </execution>
                 </executions>

